awx/main/management/commands/provision_instance.py

```diff
@@ -18,11 +18,12 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('--hostname', dest='hostname', type=str, help='Hostname used during provisioning')
+        parser.add_argument('--node_type', type=str, default="hybrid", choices=["control", "execution", "hybrid"], help='Instance Node type')
```

Comment:
@chrismeyersfsu I guess maybe execution nodes have to be registered by the installer to be added to a group?

In that case, our heartbeat tasks won't be adding the instance, they will be marking its capacity as non-zero, as if it had went dark previously.